src/content/release-notes.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.16.0
+
+20 December 2023
+
+### Improvements
+
+- Okteto garbage collector cronjob now resolves buildkit and ingress hosts to their internal IPs  <!-- 7432 -->
+- Registry Credentials cronjob only update dockerconfig secret if the credentials change.  <!-- 7474 -->
+- Okteto Daemon now update kubelet docker config.json file in nodes reactively when the static docker config secret changes. Bumps okteto/daemon to 0.5.1
+- Fix network policies configuration to limit buildkit access to the nginx controller  <!-- 7505 -->
+- Add Okteto Insights Token to the Admin UI  <!-- 7473 -->
+- Remove customer data from mixpanel events  <!-- 7390 -->
+- Updated documentation structure to make content easier to navigate and find. Release notes can now be found at [https://www.okteto.com/docs/release-notes/](https://www.okteto.com/docs/release-notes/), and chart values reference can be found at [https://www.okteto.com/docs/reference/helm-chart-values/](https://www.okteto.com/docs/reference/helm-chart-values/).  All documentation changes are listed [here](https://github.com/okteto/docs/pulls?q=is%3Apr+label%3Adoc-restructure+).
+
+### Features
+
+- Add `okteto_usage_build_duration_seconds` build metrics to okteto insights. More info [here](https://www.okteto.com/docs/1.16/admin/okteto-insights)  <!-- 7445 -->
+- When telemetry is enabled, Okteto now exports cluster configuration to Mixpanel on every helm upgrade.
+- A diagnostics tool is now provided to help troubleshoot installations. More info [here](https://www.okteto.com/docs/1.16/self-hosted/manage/diagnostics/)
+- The Okteto CLI used for Pipeline installation has been upgraded to version [2.23.1](https://github.com/okteto/okteto/releases/tag/2.23.1)
+- Move garbage collection of `okteto up` based on last synced file behind a feature flag. More info [here](https://www.okteto.com/docs/reference/helm-chart-values/#gc)   <!-- 7424 -->
```

Comment:
The link seems broken, at least I get a not found page. Given that is a link for the documentation, shouldn't it be a relative link to the helm chart configuration page?

Reply:
This is because of the change in the sidebar. Once we change the default this link will work. Some of the ones below will point to the old paths but we have redirection in place so it should be ok.

Reply:
Btw, I'll change all to relative links

Reply:
I think it's because of the new sidebar and reorg. See: https://github.com/okteto/docs/pull/569#issuecomment-1862885741

Reply:
That might be. If not, we can ask Hugues about it, but it makes sense what you mention in the comment. I remember I had some issue in the past for not including the extension in the links, that's why I mentioned, but I don't remember the details